src/rules/store/select-in-selector-names.ts

```diff
@@ -0,0 +1,76 @@
+import path from 'path'
+import {
+  AST_NODE_TYPES,
+  ESLintUtils,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils'
+import { isIdentifier, isLiteral } from '../../utils/helper-functions/index'
+
+import { memorizedSelector, selectorsNames, docsUrl } from '../../utils'
+
+export const messageId = 'selectInSelectorNames'
+export type MessageIds = typeof messageId
+
+type Options = []
+
+export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
+  name: path.parse(__filename).name,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      category: 'Best Practices',
+      description: 'Name Selector Functions as selectThing',
```

Comment:
Should we rename this rule to `selectors-start-with-select` ?
The description and the message could be `The variable name of a selector should start with "select", for example "selectThing"`?